documentation/swagger-checklist.md

```diff
@@ -247,6 +247,8 @@ For example, [NetworkInterface.ipConfigurations](https://github.com/Azure/azure-
 
 - :white_check_mark: **M2060**: x-ms-pageable operations must have a 200 response defined.
 
+- :white_check_mark: **M2061**: URLs should have parameterized resource type values instead of defaults.
```

Comment:
I think we want to put more than this in the documentation, can you write down the pattern for the URL and indicate what you're referring to as type values versus other parts of the path? Also, it'd be good to write (at least in parenthesis) the name of the rule (ProvidersPathValidation) I think it makes it easier to identify for swagger authors and reviewers (numbers are just not as easy to remember and could change). Feel free to consult with Kirthi as well if needed.